agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -142,6 +142,12 @@ function install_k8s_agent {
         echo -e "    $ADDITIONAL_CONF" >> $CONFIG_FILE
     fi
 
+    if [[ $PLATFORM ==  "Darwin" ]]; then
+        sed -i "" "s|# serviceAccount: sysdig-agent|serviceAccount: sysdig-agent|" /tmp/sysdig-agent-daemonset-v2.yaml
```

Comment:
I think that `-i -e` will work here on both platforms without the `-i ""` for darwin

Reply:
Exactly what I needed. I was struggling with this for a bit and then also tried this with an `awk` so that the one command could apply to both platforms. But then ended up with the `IF` to detect platform which looks ugly.

Just tested the change for using `sed -i -e` on MacOS and Ubuntu and both worked. Thank you so much, @mbreitung!